### ISSUE TITLE
Parse Push - production: false // Crashes - Wiki needs typo fix

### ISSUE THREAD

**Issue by gateway** [ParsePlatform/parse-server#688]
This is the first time im setting up parse push to test this out in our server environment.  I followed the wiki and added the push block in.

```
`  push: {
      ios: {
        pfx: 'keys/mykey-dev-apns.p12', // The filename of private key and certificate in PFX or PKCS12 format from disk  
        //cert: '', // If not using the .p12 format, the path to the certificate PEM to load from disk
        //key: '', // If not using the .p12 format, the path to the private key PEM to load from disk
        bundleId: 'com.myapp-dev' // The bundle identifier associate with your app
        production: false // Specifies which environment to connect to: Production (if true) or Sandbox
      }
    }`
```

When starting the server on Heroku it fails wit this log..

```
2016-02-26T18:15:49.320711+00:00 heroku[web.1]: State changed from up to starting
2016-02-26T18:15:49.317594+00:00 heroku[web.1]: Restarting
2016-02-26T18:15:50.644368+00:00 heroku[web.1]: Starting process with command `npm start`
2016-02-26T18:15:52.636004+00:00 app[web.1]: npm ERR! Linux 3.13.0-77-generic
2016-02-26T18:15:52.562648+00:00 app[web.1]: 
2016-02-26T18:15:52.562664+00:00 app[web.1]: > parse-server-example@1.2.0 start /app
2016-02-26T18:15:52.637180+00:00 app[web.1]: npm ERR! Exit status 1
2016-02-26T18:15:52.562665+00:00 app[web.1]: > node index.js
2016-02-26T18:15:52.637091+00:00 app[web.1]: npm ERR! parse-server-example@1.2.0 start: `node index.js`
2016-02-26T18:15:52.637292+00:00 app[web.1]: npm ERR! 
2016-02-26T18:15:52.637658+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the parse-server-example package,
2016-02-26T18:15:52.637434+00:00 app[web.1]: npm ERR! Failed at the parse-server-example@1.2.0 start script 'node index.js'.
2016-02-26T18:15:52.562666+00:00 app[web.1]: 
2016-02-26T18:15:52.637540+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2016-02-26T18:15:52.637938+00:00 app[web.1]: npm ERR!     node index.js
2016-02-26T18:15:52.638037+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2016-02-26T18:15:52.638371+00:00 app[web.1]: npm ERR!     npm owner ls parse-server-example
2016-02-26T18:15:52.638467+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
**2016-02-26T18:15:52.622015+00:00 app[web.1]: /app/index.js:30
2016-02-26T18:15:52.622034+00:00 app[web.1]:         production: false // Specifies which environment to connect to: Production (if true) or Sandbox**
2016-02-26T18:15:52.622037+00:00 app[web.1]: 
2016-02-26T18:15:52.622036+00:00 app[web.1]:         ^^^^^^^^^^
2016-02-26T18:15:52.641740+00:00 app[web.1]: 
2016-02-26T18:15:52.641898+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2016-02-26T18:15:52.622061+00:00 app[web.1]:     at Module.load (module.js:357:32)
2016-02-26T18:15:52.636298+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2016-02-26T18:15:52.636459+00:00 app[web.1]: npm ERR! node v5.6.0
2016-02-26T18:15:52.622063+00:00 app[web.1]:     at startup (node.js:140:18)
2016-02-26T18:15:52.622038+00:00 app[web.1]: SyntaxError: Unexpected identifier
2016-02-26T18:15:52.622039+00:00 app[web.1]:     at exports.runInThisContext (vm.js:53:16)
2016-02-26T18:15:52.622062+00:00 app[web.1]:     at Function.Module._load (module.js:314:12)
2016-02-26T18:15:52.622063+00:00 app[web.1]:     at node.js:1001:3
2016-02-26T18:15:52.641998+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2016-02-26T18:15:52.622059+00:00 app[web.1]:     at Module._compile (module.js:387:25)
2016-02-26T18:15:52.622060+00:00 app[web.1]:     at Object.Module._extensions..js (module.js:422:10)
2016-02-26T18:15:52.622062+00:00 app[web.1]:     at Function.Module.runMain (module.js:447:10)
2016-02-26T18:15:52.628137+00:00 app[web.1]: 
2016-02-26T18:15:52.636972+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2016-02-26T18:15:52.638161+00:00 app[web.1]: npm ERR!     npm bugs parse-server-example
2016-02-26T18:15:52.636771+00:00 app[web.1]: npm ERR! npm  v3.6.0
2016-02-26T18:15:52.638260+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2016-02-26T18:15:52.637746+00:00 app[web.1]: npm ERR! not with npm itself.
2016-02-26T18:15:52.637850+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2016-02-26T18:15:53.142934+00:00 heroku[web.1]: Stopping all processes with SIGTERM
2016-02-26T18:15:53.240811+00:00 heroku[web.1]: Process exited with status 1
2016-02-26T18:15:53.249719+00:00 heroku[web.1]: State changed from starting to crashed
2016-02-26T18:15:53.250857+00:00 heroku[web.1]: State changed from crashed to starting
2016-02-26T18:15:54.937802+00:00 heroku[web.1]: Starting process with command `npm start`
2016-02-26T18:15:55.085645+00:00 heroku[web.1]: Process exited with status 143
2016-02-26T18:15:57.573851+00:00 app[web.1]: 
2016-02-26T18:15:57.573861+00:00 app[web.1]: > parse-server-example@1.2.0 start /app
2016-02-26T18:15:57.693267+00:00 app[web.1]: /app/index.js:30
2016-02-26T18:15:57.573862+00:00 app[web.1]: > node index.js
2016-02-26T18:15:57.573863+00:00 app[web.1]: 
**2016-02-26T18:15:57.693272+00:00 app[web.1]:         production: false // Specifies which environment to connect to: Production (if true) or Sandbox**
2016-02-26T18:15:57.693275+00:00 app[web.1]: 
2016-02-26T18:15:57.693283+00:00 app[web.1]:     at Module.load (module.js:357:32)
2016-02-26T18:15:57.693274+00:00 app[web.1]:         ^^^^^^^^^^
2016-02-26T18:15:57.693280+00:00 app[web.1]: SyntaxError: Unexpected identifier
2016-02-26T18:15:57.693284+00:00 app[web.1]:     at Function.Module._load (module.js:314:12)
2016-02-26T18:15:57.693281+00:00 app[web.1]:     at Module._compile (module.js:387:25)
2016-02-26T18:15:57.693281+00:00 app[web.1]:     at exports.runInThisContext (vm.js:53:16)
2016-02-26T18:15:57.693285+00:00 app[web.1]:     at startup (node.js:140:18)
2016-02-26T18:15:57.693285+00:00 app[web.1]:     at Function.Module.runMain (module.js:447:10)
2016-02-26T18:15:57.693286+00:00 app[web.1]:     at node.js:1001:3
2016-02-26T18:15:57.693282+00:00 app[web.1]:     at Object.Module._extensions..js (module.js:422:10)
2016-02-26T18:15:57.703777+00:00 app[web.1]: 
2016-02-26T18:15:57.715968+00:00 app[web.1]: npm ERR! Linux 3.13.0-77-generic
2016-02-26T18:15:57.716787+00:00 app[web.1]: npm ERR! node v5.6.0
2016-02-26T18:15:57.717923+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2016-02-26T18:15:57.716492+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2016-02-26T18:15:57.718146+00:00 app[web.1]: npm ERR! parse-server-example@1.2.0 start: `node index.js`
2016-02-26T18:15:57.717653+00:00 app[web.1]: npm ERR! npm  v3.6.0
2016-02-26T18:15:57.718303+00:00 app[web.1]: npm ERR! Exit status 1
2016-02-26T18:15:57.718675+00:00 app[web.1]: npm ERR! Failed at the parse-server-example@1.2.0 start script 'node index.js'.
2016-02-26T18:15:57.718484+00:00 app[web.1]: npm ERR! 
2016-02-26T18:15:57.718930+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2016-02-26T18:15:57.719635+00:00 app[web.1]: npm ERR!     node index.js
2016-02-26T18:15:57.720263+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2016-02-26T18:15:57.720037+00:00 app[web.1]: npm ERR!     npm bugs parse-server-example
2016-02-26T18:15:57.719148+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the parse-server-example package,
2016-02-26T18:15:57.719325+00:00 app[web.1]: npm ERR! not with npm itself.
2016-02-26T18:15:57.719479+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2016-02-26T18:15:57.719840+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2016-02-26T18:15:57.720453+00:00 app[web.1]: npm ERR!     npm owner ls parse-server-example
2016-02-26T18:15:57.720623+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
2016-02-26T18:15:57.726238+00:00 app[web.1]: 
2016-02-26T18:15:57.726487+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2016-02-26T18:15:57.726682+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2016-02-26T18:15:58.503470+00:00 heroku[web.1]: Process exited with status 1
2016-02-26T18:15:58.847604+00:00 heroku[web.1]: State changed from starting to crashed
2016-02-26T18:15:59.743101+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=secret-ridge-13599.h
```

**Comment by gateway** [comment]
After digging around it seems that the wiki has a typo and missing a ',' 

for all the block snippets like this one

`  var server = new ParseServer({
    databaseURI: '...',
    cloud: '...',
    appId: '...',
    masterKey: '...',
    push: {
      android: {
        senderId: '...',
        apiKey: '...'
      },
      ios: {
        pfx: '/file/path/to/XXX.p12',
        bundleId: ''
        production: false
      }
    }
  })`

should be

```
`  var server = new ParseServer({
    databaseURI: '...',
    cloud: '...',
    appId: '...',
    masterKey: '...',
    push: {
      android: {
        senderId: '...',
        apiKey: '...'
      },
      ios: {
        pfx: '/file/path/to/XXX.p12',
        bundleId: '',
        production: false
      }
    }
  })`
```

basically missing a comma after bundleid: ''

**Comment by gateway** [comment]
Anyone? :beer:

**Comment by drew-gross** [comment]
Thanks! I've fixed the typo.
